doc/tech/approach.md

```diff
@@ -3,97 +3,226 @@
+
+Cloud and Telco networking are layered, and it is very important to keep the layering dependencies low to enable security, separation and portability in between multiple implementations.
+
+Before we start developing a deep model we need to agree on some foundational concepts and layering that allow decoupling of implementations in between the layers. We will emphasize four concepts in this section
+
+ - Underlay and Overlay Networking concepts
+ - Hardware and Software Infrastructure Layer concepts
+ - Software Defined Networking, SDNu and SDNo concepts
+ - Programmable Networking Fabric concept
+
+<a name="5.1.3.1"></a>
+#### 5.1.3.1 Underlay and Overlay Networking concepts
+
+The ETSI NFV model divides networking into an Underlay and an Overlay Network layer. The purpose with this layering is to ensure separation of the SW Virtualisation tenants (Workload) Overlay Networks from each other, whilst allowing the traffic to flow on the shared Underlay Network in between all Ethernet connected HW units.
+
+The Overlay Networking separation is often done through encapsulation of Tenants' traffic using overlay protocols e.g. through VxLAN or EVPN over the Underlay Network e.g. based on L2 (VLAN) or L3 (IP) networks.
+
+In some instances, the SW Virtualisation Tenants can bypass the Overlay Networking encapsulation to achieve better performance or network visibility/control. A common method to bypass the Overlay Networking encapsulation normally done by the SW Virtualisation Layer, is the VNF/CNF usage of SR-IOV that effectively take over the Physical and Virtual Functions of the NIC directly into the VNF/CNF Tenant. In these cases, the Underlay Networking must handle the separation e.g. through a Virtual Termination End Point (VTEP) that encapsulate the Overlay Network traffic.
+
+> **Note:** Bypassing the Overlay Networking layer is a violation of the basic CNTT decoupling principles, but in some cases unavoidable with existing technologies and standards. Until suitable technologies and standards are developed, CNTT have a set of agreed exemptions that forces the Underlay Networking to handle the bypassed Overlay Networking separation.
+
+VTEP could be manually provisioned in the Underlay Networking or be automated and controlled through a Software Defined Networking controller interfaces into the underlying networking in the HW Infrastructure Layer. 
+
+<a name="5.1.3.2"></a>
+#### 5.1.3.2 Hardware and Software Infrastructure Layer concepts
+
+The Cloud Infrastructure can be considered to be composed of two distinct layers, referred to as HW Infrastructure Layer and SW Infrastructure Layer. When there are multiple separated simultaneously deployed SW Virtualisation domains, the architecture and deployed implementations must enable each of them to be in individual non-dependent administrative domains. The HW Infrastructure must also be enabled to be a fully separated administrative domain from all of the SW Virtualisation domains. 
```

Comment:
The wording " .. can be considered composed of ..": Does it mean the Cloud Infrastructure MUST be layered? That would be a general RM requirement and also compute infrastructure and storage are layered. ETSI doesn't define this. Also having these as completely separated administrative domains should not be a requirement.

Reply:
@ulikleber, definitely 'can be" and "must" are different statements! We are saying that "the architecture and deployed implementations must enable each of them to be in individual non-dependent administrative domains." not that there must be completely separated admin domains.  Yes, perhaps ETSI does not introduce this layering.  This is one of the "gaps" we need to fill.  As said in other places, layering is on architectural level, not a deployment requirement. Hope this clarifies!

Reply:
@ulikleber, as a matter of fact in RM Ch03 the Cloud Infrastructure is presented like:
![image](https://user-images.githubusercontent.com/52221975/80552403-d91b3880-8a09-11ea-9c15-921a7c2da762.png)
So generally in RM we already distinguish two layers of Infrastructure (called there Virtual resources and Physical resources).  That means that your request to have a "general RM requirement and also compute infrastructure and storage are layered" is already satisfied. We may think of aligning terminology better but the idea of such layering is already there.